apps/user-ms/src/main/java/it/pagopa/selfcare/user/controller/UserController.java

```diff
@@ -18,6 +23,13 @@ public class UserController {
     private final UserMapper userMapper;
     private final UserEventService userEventService;
 
-
+    @Operation(summary = "The API retrieves Users' emails using institution id and product id")
+    @GET
+    @Path(value = "/emails/{institutionId}/{productId}")
```

Comment:
you should improve REST path standardization using **/institutions/{institutionId}/users/emails?productId={productId}** or **/users/emails?productId={productId}&institutionId={institutionId}**